package-manifest.toml

```diff
@@ -116,15 +116,15 @@ setup_hint = """
 - Run `pkg install library/postgresql-13` to download Postgres libraries
 """
 
-[package.oximeter-collector]
+[package.oximeter]
```

Comment:
@smklein I had to rename this because the other intermediary package seems to _have_ to be named `oximeter-collector` because of the rust package name? Do you think this will cause problems down the line?

Reply:
Chatted offline about this, TL;DR:

- The "real oximeter package" needs to be called `package.oximeter-collector` because that's the name of the rust crate we're building
- The "real oximeter package" choice of `service_name` influences where the binary in this tarball gets placed

For the **composite** package, we have no such restrictions, and can pick whatever naming conventions we want.